lib/ngsiHandlers.js

```diff
@@ -176,11 +204,12 @@ function ngsiExecuteHandler(id, type, attributes, callback) {
                 device.type,
                 device.apikey,
                 attribute.name,
-                'pruebas',
+                // Ezequiel: mandar el valor del ack como resultado de la órden.
```

Comment:
You right, I'll get rid of them, at the time I wrote this I was not thinking to publish this code.